task/anchore-cli/0.2/README.md

```diff
@@ -0,0 +1,80 @@
+# Anchore CLI
+
+The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+Anchore CLI will try to connect to the Anchore Engine at http://localhost/v1 with no authentication. The username, password and URL for the server can be passed to the Anchore CLI as command line arguments
+
+## Install the Task
```

Comment:
any reason for adding 2 versions of the same task?

Reply:
first version I had passed ANCHORE USERNAME and Password as env variables
second version refers to using secret

Reply:
Can we merge both? Also, IMHO referring via secrets would be a better option but I'll leave it upto you.

Reply:
@vinamra28  have merged

Reply:
@vinamra28 any update